distributed/dashboard/components/scheduler.py

```diff
@@ -243,12 +243,10 @@ def __init__(self, scheduler, width=600, **kwargs):
                     "proc_memory": [0] * 4,
                     "color": ["blue", "blue", "blue", "grey"],
                     "alpha": [1, 0.7, 0.4, 1],
-                    "memtype": [
-                        "managed (in memory)",
-                        "unmanaged",
-                        "unmanaged, recently increased",
-                        "managed (spilled to disk)",
-                    ],
+                    "man_in_mem": [0] * 4,
+                    "man_spilled": [0] * 4,
+                    "unman_old": [0] * 4,
+                    "unman_rec": [0] * 4,
```

Comment:
Please avoid cryptic abbreviations - bytes are cheap.
Please align these to the variable names of MemoryState.